### PR TITLE
Handle paginated results from the Github API

### PR DESCRIPTION
**Pull request by aswanson-nr**
# Summary
Update `add-files-to-translation-queue.js` to handle paginated results from the Github API and abstract that logic.

### Closes https://github.com/newrelic/docs-website/issues/2727

### CODE REVIEW

File: scripts/actions/utils/__tests__/github-api-helpers.test.js [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+const fetch = require('node-fetch');
+
+describe('Github API Helpers', () => {
+  afterEach(() => {
+    jest.resetAllMocks();
+  });
+
+  describe('getNextLink', () => {
+    test('returns null for malformed header', async () => {
```
Comment (nr-kkenney) [inline scripts/actions/utils/__tests__/github-api-helpers.test.js:18]: im curious why these tests are async

File: scripts/actions/utils/__tests__/github-api-helpers.test.js [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+      const page = await fetchPaginatedGHResults('test', 'test');
+      expect(page).toEqual(files);
+    });
+
+    test('returns more than 1 page of results', async () => {
+      const files = [{ file: 1 }, { file: 2 }, { file: 3 }];
+      fetch
+        .mockResolvedValueOnce({
+          ok: true,
+          json: () => Promise.resolve(files.slice(0, 1)),
```
Comment (nr-kkenney) [inline scripts/actions/utils/__tests__/github-api-helpers.test.js:78]: im curious why you went with slice over files[0], files[1], etc.

**Review by nr-kkenney — APPROVED** [review]
only had a couple questions which i dont think impact the functionality, so approving. feel free to answer em or skip em.

File: scripts/actions/utils/__tests__/github-api-helpers.test.js (hunk 1 above)
Comment (aswanson-nr) [reply scripts/actions/utils/__tests__/github-api-helpers.test.js]: Whoops! copy paste error

File: scripts/actions/utils/__tests__/github-api-helpers.test.js (hunk 2 above)
Comment (aswanson-nr) [reply scripts/actions/utils/__tests__/github-api-helpers.test.js:78]: It was just an easy way to keep them in an array instead of a single item.

**Review by nr-kkenney — APPROVED** [review]

**Pull request merged**